CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+**12.6.6** (2025-12-17)
```

Comment:
issue: would be 13.0.0 with breaking changes, right? or is the rest still there and it's not being dropped yet? Then I think it'll be a minor release